src/platform/workers/sync/SyncWorker.ts

```diff
@@ -1,13 +1,29 @@
 import {SharedWorker} from "../SharedWorker";
-import {StartSyncRequest, StartSyncResponse, SyncRequestType} from "../types/sync";
+import {StartSyncRequest, StartSyncResponse, SyncEvent, SyncRequestType, SyncStatusChanged} from "../types/sync";
+import {Event, makeEventId} from "../types/base";
 
 export class SyncWorker extends SharedWorker {
+    private readonly _eventBus: BroadcastChannel;
+
     constructor() {
         super();
+        this._eventBus = new BroadcastChannel(this.name);
         this.setHandler(SyncRequestType.StartSync, this.startSync.bind(this));
     }
 
     async startSync(request: StartSyncRequest, response: StartSyncResponse): Promise<StartSyncResponse> {
+        const event: SyncStatusChanged = {
+            id: makeEventId(),
+            type: SyncEvent.StatusChanged,
+            data: {
+                newValue: "Stopped",
+            }
+        }
+        this.broadcastEvent(event);
```

Comment:
I take it, for the scope of this PR, the idea is to broadcast that sync event has finished without doing it, right? `@TODO` would have made it obvious at a glance :)